src/components/organisms/UiHorizontalPaging/UiHorizontalPaging.vue

```diff
@@ -286,6 +277,23 @@ const menuItemsSlots = computed(() => (Object.keys(slots).reduce((object, slotNa
   }
   return object;
 }, {})));
+const internalMenuTemplateRefs = ref<InstanceType<typeof UiMenu> | null>(null);
+watch(() => props.menuTemplateRefs, () => {
+  internalMenuTemplateRefs.value = props.menuTemplateRefs;
+});
+const handleBackClick = () => {
+  activeItems.value = activeItems.value.slice(0, -1);
+  if (internalMenuTemplateRefs.value?.lastFocusedMenuItemTemplateRefs) {
+    focusElement(internalMenuTemplateRefs.value?.lastFocusedMenuItemTemplateRefs.itemTemplateRefs.content.$el, true);
```

Comment:
As we discussed previously, there is good chance that used slot would have different internal structure. At the very least we should use conditional chaining everywhere.

Reply:
IMO conditional changing sounds like the solution for this ISSUE. We should handle common cases. Users can use `slots` and put anything they want into it. But in this case, it should handle "this" functionally.